docs/install/native-install/ubuntu.rst

```diff
@@ -86,6 +86,10 @@ Add the ROCm repository.
 Installing
 ================================================
 
+.. important::
```

Comment:
@dbieleck Do you like where @alexxu-amd added it in the "Installing" part?  Or do you think it should be right at the top of the page?

I think I'd like it right up at line 8, as the first thing they see, even before they start "registering the repositories".

Reply:
I agree.  It might be better up at line 8 so it's visible right away.